### PR TITLE
GET EKS external cluster details

### LINKED ISSUE #8998: GET EKS external cluster details
### What would you like to be added? / User Story
GET EKS external cluster details

### PR DESCRIPTION
**Pull request by imharshita**
Signed-off-by: Harshita sharma <harshita.sharma6174@gmail.com>

**What does this PR do / Why do we need it**: GET AKS external cluster details

**Does this PR close any issues?**:
Fixes #8998

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: pkg/handler/v2/external_cluster/eks.go [hunk 1]
```diff
@@ -413,3 +413,40 @@ func deleteEKSNodeGroup(cluster *kubermaticv1.ExternalCluster, nodeGroupName str
...
+		apiCluster.Cloud.EKS.ClusterSpec = &clusterSpec
+		return apiCluster, nil
+	}
+	clusterSpec.ResourcesVpcConfig = apiv2.VpcConfigRequest{
+		SecurityGroupIds: eksCluster.ResourcesVpcConfig.SecurityGroupIds,
+		SubnetIds:        eksCluster.ResourcesVpcConfig.SubnetIds,
+	}
+	apiCluster.Cloud.EKS.ClusterSpec = &clusterSpec
+
+	return apiCluster, nil
```
Comment (lsviben) [inline pkg/handler/v2/external_cluster/eks.go]: NIT: a suggestion to make it a bit simpler

```suggestion
	clusterSpec := &apiv2.EKSClusterSpec{}
	apiCluster.Cloud.EKS.ClusterSpec = clusterSpec
	eksCluster := clusterOutput.Cluster
	if eksCluster == nil {
		return apiCluster, nil
	}
	
	clusterSpec.RoleArn = aws.StringValue(eksCluster.RoleArn)
	clusterSpec.Version = aws.StringValue(eksCluster.Version)

	if eksCluster.ResourcesVpcConfig != nil {
		clusterSpec.ResourcesVpcConfig = apiv2.VpcConfigRequest{
			SecurityGroupIds: eksCluster.ResourcesVpcConfig.SecurityGroupIds,
			SubnetIds:        eksCluster.ResourcesVpcConfig.SubnetIds,
		}
	}

	return apiCluster, nil
```

Comment (imharshita) [reply pkg/handler/v2/external_cluster/eks.go]:
```
eksCluster := clusterOutput.Cluster
	if eksCluster == nil {
		return apiCluster, nil
	}

	clusterSpec := &apiv2.EKSClusterSpec{
		RoleArn: aws.StringValue(eksCluster.RoleArn),
		Version: aws.StringValue(eksCluster.Version),
	}

	if eksCluster.ResourcesVpcConfig != nil {
		clusterSpec.ResourcesVpcConfig = apiv2.VpcConfigRequest{
			SecurityGroupIds: eksCluster.ResourcesVpcConfig.SecurityGroupIds,
			SubnetIds:        eksCluster.ResourcesVpcConfig.SubnetIds,
		}
	}

	apiCluster.Cloud.EKS.ClusterSpec = clusterSpec
	return apiCluster, nil
```
wdyt about this?

**Pull request merged**